File: AnnoyWechat/app/src/main/java/com/whu/annoywechat/MainActivity.java
package com.whu.annoywechat;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = (Button)
        Intent intent = new Intent(Intent.ACTION_VIEW);
        intent.setData(Uri.parse("http://www.baidu.com"));
        startActivity(intent);


    }
}

File: WechatHook/app/src/main/java/com/rliget/wechathook/MainActivity.kt
package com.rliget.wechathook

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (!isModuleActive()) {
            Toast.makeText(this, "模块未启动", Toast.LENGTH_LONG).show()
        } else {
            Toast.makeText(this, "模块已启动", Toast.LENGTH_LONG).show()
        }
    }

    private fun isModuleActive(): Boolean {
        return false
    }
}
